gradle/validation/jar-checks.gradle

```diff
@@ -333,9 +333,14 @@ configure([project(":solr"), project(":lucene"),]) {
     }
 
     doFirst {
-      def allReferenced = validationTasks.collectMany { task ->
-        task.project.jarInfos.collectMany { it.referencedFiles }
-      }.collect { it.toString() }
+      def allReferenced = validationTasks
+          .findAll { task ->
```

Comment:
Can you add a commend linking to the issue and perhaps shorten to:
{code}
// Only collect for enabled tasks: https://issues.apache.org/jira/browse/LUCENE-9780
.findAll { it.enabled }
.collectMany...
{code}